gm4_orb_of_ankou/beet.yaml

```diff
@@ -3,7 +3,10 @@ name: Orb of Ankou
 version: 1.7.X
 
 data_pack:
-  load: .
+  load: [
+    .,
+    https://cdn.modrinth.com/data/oDhxdGVZ/versions/aW4ph2OC/player_motion.zip
+  ]
```

Comment:
You should format the yaml array like this imo:
```yaml
load:
  - '.'
  - 'https://cdn.modrinth.com/data/oDhxdGVZ/versions/aW4ph2OC/player_motion.zip'
```

Reply:
true; I forgot how to properly format yaml